libappstream-glib/as-release.c

```diff
@@ -769,6 +774,67 @@ as_release_set_description (AsRelease *release,
 			     as_ref_string_new (description));
 }
 
+/**
+ * as_release_get_urls:
+ * @release: a #AsRelease instance.
+ *
+ * Gets the URLs set for the application.
+ *
+ * Returns: (transfer none): hash table of URLs
+ *
+ * Since: 0.7.15
+ **/
+GHashTable *
+as_release_get_urls (AsRelease *release)
```

Comment:
Is this actually needed as public API? Seems to leak a lot of internal details (hash table).

Reply:
I just coped the design (mistake?) from AsApp. I can nuke this, I'm not using it at the moment. Ideas welcome.

Reply:
I think I'd leave it out then and add it only when something actually needs it.

Reply:
Okay, agree. Thanks.